packages/apps/job-launcher/server/src/common/enums/content-moderation.ts

```diff
@@ -0,0 +1,18 @@
+export enum ContentModerationRequestStatus {
+  PENDING = 'pending',
+  PROCESSED = 'processed',
+  POSSIBLE_ABUSE = 'possible_abuse',
+  POSITIVE_ABUSE = 'positive_abuse',
+  PASSED = 'passed',
+  FAILED = 'failed',
+}
+
+export enum ContentModerationLevel {
```

Comment:
It is GCV specific specify it in the name and might be worth to move this and ContentModerationFeature to a file constants/GCV